packages/blade/src/components/Tooltip/TooltipInteractiveWrapper.native.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import type { PressableProps, View } from 'react-native';
+import { Pressable } from 'react-native';
+import styled from 'styled-components/native';
+
+const StyledPressable = styled(Pressable)(() => {
+  return { alignSelf: 'flex-start' };
+});
+
+const TooltipInteractiveWrapper = React.forwardRef<View, PressableProps>((props, ref) => {
+  return (
+    <StyledPressable ref={ref} collapsable={false} {...props}>
```

Comment:
so with this our non-interactive icons will now become pressable? and the default styling props of pressable will also apply?

Reply:
Yep. Don't think Pressable has any default styling. It's a blank slate.

Reply:
when you press it it'll show that pressable effect. can you verify once? and also now we are actually doing the same thing you don't intend to do as [mentioned here](https://github.com/razorpay/blade/pull/1265#discussion_r1227942459)

Reply:
ReactNative is a different case, in react native Tooltip opens via click and works more like a "popover" thus it will have to be clickable.